okhttp/src/nonJvmMain/kotlin/okhttp3/internal/-NormalizeNonJvm.kt

```diff
@@ -13,5 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package okhttp3.internal
+
+import com.squareup.okhttpicu.SYSTEM_NORMALIZER
+
+internal actual fun normalizeNfc(string: String): String =
```

Comment:
I avoided depending on okhttp-icu from the main OkHttp module. The JVM gives us everything we need and I’m reluctant to add an entire jar dependency for something that’s a pass-through to this method.